src/build.ts

```diff
@@ -73,6 +73,10 @@ export async function build(
     effects.output.write(`${faint("parse")} ${sourcePath} `);
     const start = performance.now();
     const page = await parseMarkdown(sourcePath, options);
+    if (page?.data?.draft) {
+      effects.logger.log(yellow("skipping [draft]"));
```

Comment:
```suggestion
      effects.logger.log(faint("(skipped)"));
```

This would match what we do for search indexing.